### PR TITLE
ZmqPusher works with other protocols than tcp

### PR DESCRIPTION
**Pull request by SimonHeimberg**

### CODE REVIEW

File: Pusher/Zmq/ZmqServerPushHandler.php [hunk 1]
```diff
@@ -91,7 +91,7 @@ public function handle(LoopInterface $loop, WampServerInterface $app)
                 $this->eventDispatcher->dispatch(Events::PUSHER_SUCCESS, new PushHandlerEvent($data, $this));
             } catch (\Exception $e) {
                 $this->logger->error(
-                    'AMQP handler failed to ack message', [
+                    'zmq handler failed to ack message', [
```
[the remark this answers is not in the document]
Comment (SimonHeimberg) [reply Pusher/Zmq/ZmqServerPushHandler.php]: Sorry, ci checker, but this is not my fault. ;-)

Shall I do some cleanup anyway?

[the remark this answers is not in the document]
Comment (ProPheT777) [reply Pusher/Zmq/ZmqServerPushHandler.php]: Nope it's fine ;)

**Comment by ProPheT777** [comment]
thank you @SimonHeimberg

**Comment by ProPheT777** [comment]
Seems I was on the way from the config 

```
                ->enumNode('protocol')
                    ->defaultValue('tcp')
                    ->values(['tcp', 'ipc', 'inproc', 'pgm', 'epgm'])
                ->end()
```

And finally hard code it .... ¯\_(ツ)_/¯

**Pull request merged**